torchvision/prototype/transforms/_deprecated.py

```diff
@@ -18,7 +17,7 @@
 class ToTensor(Transform):
 
     # Updated transformed types for ToTensor
-    _transformed_types = (torch.Tensor, features._Feature, PIL.Image.Image, np.ndarray)
+    _transformed_types = (is_simple_tensor, features._Feature, PIL.Image.Image, np.ndarray)
```

Comment:
Should we have `features._Feature` here? For `Image`'s it would unwrap IIUC, which is not what want. For all other features, this transform will fail, no?

Plus, having `is_simple_tensor` *and* `features._Feature` in this list is equivalent to `torch.Tensor`.

Reply:
I think @datumbox wanted to fix all incorrect `_transformed_types` usages and remove if/else inside `_transform`. So, here I just replace `torch.Tensor` by `is_simple_tensor`

Reply:
You are right. I will follow up on a PR to update these. The scope of this PR is just to change the `torch.Tensor` with `is_simple_tensor`.

Reply:
Ah, my bad.